src/Microsoft.SqlTools.ServiceLayer/EditData/EditColumnWrapper.cs

```diff
@@ -19,11 +19,23 @@ public class EditColumnWrapper
         public DbColumnWrapper DbColumn { get; set; }
 
         /// <summary>
+        /// If set, this is a string representation of the default value. If set to null, then the
```

Comment:
The logic basically generates a list of display values for the new row in the UI.
* No default is returned as `null`
* `NULL` default is returned as "NULL" (as per how SMO returns the value)
* Literal defaults are unwrapped from what SMO gives us and returned (eg, `((8))` => 8, `N'stuff'` => stuff)
* Computed columns are returned as a placeholder